sdk/storage/storage-blob-changefeed/package.json

```diff
@@ -94,7 +94,7 @@
     ]
   },
   "dependencies": {
-    "@azure/storage-blob": "^12.6.0-beta.1",
+    "@azure/storage-blob": "12.6.0",
```

Comment:
Because storage-blob-changefeed is in preview, and is not part of the packages we will hotfix, we are making the minimal number of changes we can to this package. That involves pinning it against the last known compatible storage-blob package to avoid having to update our core-tracing dependency here.